pages/windows/if.md

```diff
@@ -0,0 +1,24 @@
+# if
+
+> Performs conditional processing in batch programs.
+> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/if>.
+
+- Check whether `%errorlevel%` is greater than or equal to specified value:
+
+`if errorlevel {{exit_code}} {{command}}`
```

Comment:
Is `exit_code` the specified value you're comparing too?

Reply:
Yes.

Reply:
Then we sholud probably change the description to
``- Check whether `%errorlevel%` is greater than or equal to the specified exit code:``